client/components.html

```diff
@@ -1,16 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="./components/components.css">
+  <title>UI Components</title>
 </head>
+
 <body>
-  <ul>
-    <li><a href="./components/button/button">Button</a></li>
-    <li><a href="./components/input/input">Input</a></li>
+  <h2 class="heading-h2">Components</h2>
+  <ul class="heading-h5">
     <li><a href="./components/typography/typography.html">Typography</a></li>
+    <li><a href="./components/button/button.html">Button</a></li>
+    <li><a href="./components/input/input.html">Input</a></li>
+  </ul>
+  <h2 class="heading-h2">Forms</h2>
+  <ul class="heading-h5">
+    <li><a href="./components/Registration-Page/register.html">Registration page</a></li>
```

Comment:
If this is complete page, no need to put it in components. Just put it at the same level as index.html
Then, if you keep it's name `register.html`, after you start server you can access it by path `/register`